newsfragments/674.misc.rst

```diff
@@ -0,0 +1 @@
+Calling :func:`str` on a :exc:`trio.Cancelled` exception object returns "Cancelled" instead of an empty string.
```

Comment:
One of the tests failed on this line (https://travis-ci.org/python-trio/trio/jobs/528278782). I was apparently wrong about `:func:` being correct here (probably because `str` is also a type), and neither I nor @njsmith knows what it should be, so let's just write this as
```
``str``
```
(which won't link) for simplicity.